nats/src/main/java/io/micronaut/nats/connect/NatsConnectionFactoryConfig.java

```diff
@@ -424,4 +464,101 @@ private SSLContext createTlsContext() throws IOException, GeneralSecurityExcepti
         }
 
     }
+
+    /**
+     * Manages the jetstream configuration.
+     *
+     * @author Joachim Grimm
+     * @since 4.0.0
+     */
+    @Introspected
```

Comment:
`@Introspected` shouldn't be necessary

Reply:
This was needed for the generation of the documentation, but now it works without the annotation.